docs/source/models.rst

```diff
@@ -922,7 +922,89 @@ and behavior:
 .. |eval_importance_img| image:: _static/images/model_evaluation_feature_importance.png
    :width: 69%
 
-More information about model evaluation behaviors and outputs is available in the
+
+Evaluating on Custom Metrics
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If the default set of metrics is insufficient, you can specify a list of functions to the ``custom_metrics``
+argument of :py:func:`mlflow.evaluate()` to evaluate your models in ways that you see fit. Custom metric
+functions should accept two arguments: a DataFrame containing ``prediction`` and ``target`` columns, 
+and a dictionary of the default set of metrics. For a full list of default metrics, refer to the documentation 
+of :py:func:`mlflow.evaluate()`.
+
+The following `short example from the MLflow GitHub Repository
+<https://github.com/mlflow/mlflow/blob/master/examples/evaluation/evaluate_with_custom_metrics.py>`_ 
+uses :py:func:`mlflow.evaluate()` with a custom metric function to evaluate the performance of a regressor on the
+`California Housing Dataset <https://www.dcc.fc.up.pt/~ltorgo/Regression/cal_housing.html>`_.
+Note that custom metric functions can return both metrics and artifacts. It can either return a single
+dictionary of metrics, or two dictionaries representing metrics and artifacts.
```

Comment:
The first sentence uses a plural (`custom metric functions`) and the second sentence uses a singular pronoun (`it`). Can we make these agree?

Reply:
Will change to plural. Thanks!